parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -106,8 +106,13 @@ ERR_SYSTEMCTL_MASK_FAIL=2 # Service could not be masked by systemctl
 
 ERR_CREDENTIAL_PROVIDER_DOWNLOAD_TIMEOUT=205 # Timeout waiting for credential provider downloads
 
-OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
-OS_VERSION=$(sort -r /etc/*-release | gawk 'match($0, /^(VERSION_ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }' | tr -d '"')
+SORTED_RELEASE=""
+sortEtcRelease() {
+    SORTED_RELEASE=$(sort -r /etc/*-release)
```

Comment:
I have to turn this into a function so that I can mock it for unit test to solve Mac compatibility issue